### PR TITLE
OCPQE-18599: add workflow azure-ipi-marketplace-minimal-permission

### PR DESCRIPTION
**Pull request by jinyunma**
change job `azure-ipi-marketplace` to `azure-ipi-marketplace-minimal-permission` on 4.14+, to cover test case OCP-68606

### CODE REVIEW

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-azure-ipi-marketplace-minimal-permission-f28

File: ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/marketplace/minimal-permission/cucushift-installer-rehearse-azure-ipi-marketplace-minimal-permission-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+workflow:
+  as: cucushift-installer-rehearse-azure-ipi-marketplace-minimal-permission
+  steps: 
+    pre:
+    - chain: cucushift-installer-rehearse-azure-ipi-marketplace-minimal-permission-provision
+    post:
+    - chain: cucushift-installer-rehearse-azure-ipi-deprovision
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/marketplace/minimal-permission/cucushift-installer-rehearse-azure-ipi-marketplace-minimal-permission-workflow.yaml:7]: We need to deprovision the created minimal permission sp, right?

Comment (jinyunma) [reply ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/marketplace/minimal-permission/cucushift-installer-rehearse-azure-ipi-marketplace-minimal-permission-workflow.yaml]: good catch, fixed.

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.15-installer-rehearse-azure

File: ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/marketplace/minimal-permission/provision/cucushift-installer-rehearse-azure-ipi-marketplace-minimal-permission-provision-chain.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+chain:
+  as: cucushift-installer-rehearse-azure-ipi-marketplace-minimal-permission-provision
+  steps:
+  - ref: azure-provision-service-principal-minimal-permission
+  - chain: ipi-conf-azure
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/marketplace/minimal-permission/provision/cucushift-installer-rehearse-azure-ipi-marketplace-minimal-permission-provision-chain.yaml:5]: we can replace the following steps with `cucushift-installer-rehearse-azure-ipi-marketplace-provision`? Or even, we can skip to create the provision chain, but place them in install workflow yaml.

Comment (jinyunma) [reply ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/marketplace/minimal-permission/provision/cucushift-installer-rehearse-azure-ipi-marketplace-minimal-permission-provision-chain.yaml]: make sense, updated.

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.15-installer-rehearse-azure

**Comment by jianlinliu** [comment]
cc @liangxia to review.

**Pull request merged**